cartography/intel/aws/route53.py

```diff
@@ -295,27 +114,28 @@ def transform_record_set(record_set: Dict, zone_id: str, name: str) -> Optional[
         else:
             # this is a real A record
             # loop and add each value (IP address) to a comma separated string
-            # don't forget to trim that trailing comma!
-            # TODO can this be replaced with a string join?
-            value = ""
-            for a_value in record_set["ResourceRecords"]:
-                value = value + a_value["Value"] + ","
```

Comment:
Yeah so this doesn't make any sense but I will leave that for a follow-up PR since I've already gone a little bit past a true refactor.

The `value` field is used in analysis jobs to help determine what DNS records point to. Oftentimes this value is queried for without any parsing, so if this happens to be a comma separated list, those queries will always fail.

The fix will be to just remove the `value` field altogether and then draw from the node to the IP addresses/other records like a graph